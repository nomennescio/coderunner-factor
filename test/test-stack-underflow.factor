! Copyright 2019 nomennescio

USING: kernel tools.testest ;
IN: tests

: run-tests ( -- )

"stack underflow" describe#{
  "single test" it#{
    <{ drop -> 0 }>
  }#
  "double test" it#{
    <{ drop -> 0 }>
    <{ drop drop -> 0 }>
  }#
  "double test" it#{
    <{ drop -> 0 0 }>
    <{ drop drop -> 0 0 }>
  }#
  "double test" it#{
    <{ drop -> 0 0 }>
    <{ drop drop -> 0 }>
  }#
}#

;

MAIN: run-tests

