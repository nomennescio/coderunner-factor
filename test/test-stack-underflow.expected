
<DESCRIBE::>stack underflow

<IT::>single test

<FAILED::>Test Failed : Data stack underflow


<COMPLETEDIN::>0.052173 ms

<IT::>double test

<FAILED::>Test Failed : Data stack underflow


<FAILED::>Test Failed : Data stack underflow


<COMPLETEDIN::>0.020681 ms

<IT::>double test

<FAILED::>Test Failed : Data stack underflow


<FAILED::>Test Failed : Data stack underflow


<COMPLETEDIN::>0.013233 ms

<IT::>double test

<FAILED::>Test Failed : Data stack underflow


<FAILED::>Test Failed : Data stack underflow


<COMPLETEDIN::>0.012404 ms

<COMPLETEDIN::>0.314743 ms
